### PR TITLE
DOC-6844 update CC left-nav Tools

### PR DESCRIPTION
**Pull request by andf-crl**
Addresses: DOC-6844

- Replaced mentions of `**Monitoring**` page in CC (both Dedicated and Serverless) with new `**Tools**`.
	- Did not update instances that appeared in release notes for the DB (but did for cloud) - let me know if you disagree!!
	- Did not generally refer to the new `**Monitoring**` header, except in one place where it made sense.
	- Sent this back to v21.2 for left-nav updates, and even earlier for `cockroachcloud` docs that reference the renamed page (such as `upgrade-to-v20.1.md`). No way around this.
- Added redirect from `monitoring-page.html` to `tools-page.html` as a 301.

Staging

[tools-page.md](https://deploy-preview-16183--cockroachdb-docs.netlify.app/docs/cockroachcloud/tools-page.html)

### CODE REVIEW

File: cockroachcloud/tools-page.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-title: Monitoring Page
-summary: How to use the Monitoring page to enable monitoring with Datadog and access the DB Console.
+title: Tools Page
+summary: How to use the Tools page to enable monitoring with Datadog and access the DB Console.
 toc: true
 docs_area: manage
 ---
 
-The **Monitoring** page is accessible on {{ site.data.products.dedicated }} clusters. This page allows you to:
+The **Tools** page is accessible on {{ site.data.products.dedicated }} clusters. This page allows you to:
```
Comment (gemma-shay) [inline cockroachcloud/tools-page.md:8]: It's also only available for clusters running v22.2 or later. I think we should call that out but I don't know if we have a precedent of doing that in the Cloud docs... I'll look around. What do you think?

**Review by gemma-shay — APPROVED** [review]
I had one comment I deleted because I was just getting confused with the Jobs page, this LGTM!

**Comment by andf-crl** [comment]
@gemma-shay Awesome! Thanks so much for the review!!

**Review by nickvigilante — APPROVED** [review]
Approved `_redirects` only. If you need a complete review, let me know!

**Comment by gemma-shay** [comment]
FYI sidenav changes are ready to go out this week if this PR is ready to merge

**Comment by andf-crl** [comment]
@gemma-shay it is, I can merge whenever CC releases. Where do you usually watch for CC release notifications?

**Pull request merged**